vizier/backend/src/info/vizierdb/commands/plot/LineChart.scala

```diff
@@ -74,8 +74,9 @@ object LineChart extends Command
             xIndex      = series.get[Int](PARAM_X),
             yIndex      = series.get[Int](PARAM_Y),
             filter      = series.getOpt[String](PARAM_FILTER),
-            sort        = true,
-            name        = series.getOpt[String](PARAM_LABEL),
+            isBarChart  = false,
```

Comment:
Please make `isBarChart`'s default value `false`.  That way, you can remove it from everywhere else.  It should only be getting set by `BarChart`, so this should not increase confusion.

Why is `sort` being changed to `false` here?